ts/torch_handler/base_handler.py

```diff
@@ -69,37 +62,61 @@ def initialize(self, context):
         # Read the mapping file, index to object name
         mapping_file_path = os.path.join(model_dir, "index_to_name.json")
 
-        if os.path.isfile(mapping_file_path):
-            with open(mapping_file_path) as f:
-                self.mapping = json.load(f)
-        else:
-            logger.warning('Missing the index_to_name.json file. Inference output will not include class name.')
+        # Load class mapping for classifiers
+        mapping_file_path = os.path.join(model_dir, "index_to_name.json")
+        self.mapping = load_label_mapping(mapping_file_path)
 
         self.initialized = True
 
+    def _load_torchscript_model(self, model_pt_path):
+        return torch.jit.load(model_pt_path)
+
+    def _load_pickled_model(self, model_dir, model_file, model_pt_path):
+        model_def_path = os.path.join(model_dir, model_file)
+        if not os.path.isfile(model_def_path):
+            raise RuntimeError("Missing the model.py file")
+
+        module = importlib.import_module(model_file.split(".")[0])
+        model_class_definitions = list_classes_from_module(module)
+        if len(model_class_definitions) != 1:
+            raise ValueError("Expected only one class as model definition. {}".format(
+                model_class_definitions))
+
+        model_class = model_class_definitions[0]
+        state_dict = torch.load(model_pt_path)
+        model = model_class()
+        model.load_state_dict(state_dict)
+        return model
+
+
+
     def preprocess(self, data):
         """
         Override to customize the pre-processing
-        :param data: input data
-        :return: input data
+        :param data: Python list of data items
+        :return: input tensor on a device
         """
-        return data
+        return torch.as_tensor(data, device=self.device)
 
-    def inference(self, data):
+    def inference(self, data, *args, **kwargs):
```

Comment:
At least from the `handle` method in this class, args/kwargs are not getting used. What is the use case.. via overriding handle itself?